Pipfile

```diff
@@ -0,0 +1,21 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
```

Comment:
I'm _guessing_ that the `packages` vs `dev-packages` separation here is around which packages get installed when consuming vs developing this package? If so, I literally don't believe that this package has no run-time dependencies.

Reply:
Note: if this listing is secondary to what's in the `setup.py` file, then I seriously question how Pipenv is meant to work -- duplicating our dependency listing like this is a _bad_ idea.

Reply:
These packages are just for setting up the virtual environment. Installing the `sb-vision` package installs the dependencies listed in `setup.py` so we don't need to duplicate them in the `Pipfile`.

Reply:
I concluded that `sb-vision` should live in `dev-packages` simply because it is what users will be developing, but it is slightly odd.

Reply:
I'm not sure I follow. If this file is (only) used for setting up the (development) virtual environment, then surely that opens the door to the dev environment getting out of step with the deployment environment.